src/views/pages/come-again.hbs

```diff
@@ -1,2 +1,3 @@
+<link rel="stylesheet" type="text/css" href="css/post-session.css">
```

Comment:
okay all these script are in the body, not the head..

I think just mostly this is for setting the body background colour.

Instead you should give each page a top level wrapper, the colour the wrapper in your main css file. Does this make sense?